docs/content/integrations/dbt/reference.mdx

```diff
@@ -53,10 +54,10 @@ For a step-by-step implementation walkthrough, refer to the [Using dbt with Dags
   to see this concept in context.
 </Note>
 
-You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface.
+You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface. It is recommended to use the `--use-dbt-project` flag to use [DbtProject](/\_apidocs/libraries/dagster-dbt#dagster_dbt.DbtProject).
```

Comment:
```suggestion
You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface. We recommend using the `--use-dbt-project` flag to use <PyObject object="DbtProject" module="dagster_dbt" />.
```

Some nitpicky wording, but also suggesting we use the `PyObject` to autolink to the API docs.